server/repository/src/db_diesel/report_row.rs

```diff
@@ -180,3 +164,39 @@ impl Upsert for ReportRow {
         )
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
For future to make sure we don't break this.

We should have theses test on all enums

Reply:
Would be good to share this as an example of the wiki maybe? Then we can easily point to it as a how to next time this comes up?

Reply:
Thanks, good idea: https://github.com/msupply-foundation/open-msupply/wiki/Unit-Test